### DISCUSSION TITLE
Feedback from Joe!

### DISCUSSION THREAD

**Discussion by TMFukushima** [excalidraw/excalidraw#6157] [General]
After recommending the use of Exalidraw in the company's slack, my friend at work (aka Joe!) informed me that he has been using the service for many years and recommends it!

As a UI/UX designer, I really like the interface but, considering the long experience of use and having followed the tool over the years, Joe (https://github.com/dolma) gave me his Feedback and I really believe that the points mentioned can help Excalidraw!

Joe's Feedback is below:

"_Yeah the UI itself is visually cleaner, and they added libraries which is pretty nice, but I liked having everything accessible right away without having to go into menus and modals. Also their algorithm for line sloppiness used to be better, now it gets a bit buggy, especially at smaller scales. But that’s not really their fault, it’s roughJS_"

Thank you guys for the nice application!

**Comment by dwelle** [comment]
Thanks!

We may reintroduce some UI buttons that we've hidden under the menu, but not too many. Specifically the export-to-image.

Unclear what the sloppiness comment relates to though, as there shouldn't have been any changes 🤔

**Comment by TMFukushima** [comment]
@dwelle thank you for the reply. 
Yes, from my end everything is going very smoothly, but I'm using Windows 11 / Chrome (no ad blocks). I'll investigate and reply here as soon as possible. I want to recommend the Excalidraw to the designer's team. We have some "brown bag section" every week. I want to make some time to present and ask them for some thoughts. 

If you care to know about our company, please visit https://scoville.jp/
We're based in Tokyo / Japan and we work with HR and AI products along with Web Services.
We're always looking for talents as well!
